contracts/libraries/NFTSVG.sol

```diff
@@ -54,7 +54,12 @@ library NFTSVG {
                     ),
                     generateSVGCardMantle(params.quoteTokenSymbol, params.baseTokenSymbol, params.feeTier),
                     generageSvgCurve(params.tickLower, params.tickUpper, params.tickSpacing, params.overRange),
-                    generateSVGPositionDataAndLocationCurve(params.tokenId, params.tickLower, params.tickUpper),
+                    generateSVGPositionDataAndLocationCurve(
+                        params.tokenId.toString(),
```

Comment:
probably should do this toString in the method

Reply:
I use it twice in the function, so it's nice to just have it as a string right in the calldata?

Reply:
nbd, but if we were going to call this onchain it would be cheaper to pass a uint and then store the converted string in a local variable

Reply:
it brought gas up by a whopping 13 on the view.